projects/cashmere-examples/src/lib/chip-action/chip-action-example.ts

```diff
@@ -0,0 +1,14 @@
+import {Component} from '@angular/core';
+
+/**
+ * @title Action Chips
+ */
+@Component({
+    selector: 'chip-action-example',
+    templateUrl: 'chip-action-example.html'
+})
+export class ChipActionExample {
+    hideChip(event: any) {
+        event.target.style.display = 'none';
```

Comment:
Doesn't an action chip already hide itself? I believe that's what the API docs say

Reply:
nah, it just puts the 'X' and adds the pointer cursor to the chip.  I figured the hiding was something the developer would want to do themselves so they could remove it from filters at the same time.  If you think it would be useful to change that - let's add an issue

Reply:
makes sense